types/log4js.d.ts

```diff
@@ -41,7 +43,7 @@ export function connectLogger(
     statusRules?: any[];
     context?: boolean;
   }
-): any; // express.Handler;
+): express.Handler
```

Comment:
Just for consistency with line 21, the trailing semi-colon should be there.
```suggestion
): express.Handler;
```